### ISSUE TITLE
View current/latest version button on the stage

### ISSUE THREAD

**Issue by dgrumieaux** [tsgrp/hpi#951]
If a user is looking at a past version of a document, it would be nice for users to (1) know that they are looking at the non-current/non-latest version of a document, and (2) have a shortcut button that takes them to the current/latest version of a document. This would help reduce confusion for users to understand which version of the document you're on.

**Comment by gsteimer** [comment]
@dgrumieaux - let's definitely talk about this before we implement.  I could definitely see some back and forth on this one, especially if it should work in non-compliance solution scenarios.  Even in the compliance solution, what a user would want to see may be different than who they are.  For example, normal consumer users should have some sort of indication if the version they're looking at is not the latest Effective. However, doc authors may want to be notified if the version they're looking at isn't the latest Effective _or_ the latest Draft.

**Comment by dgrumieaux** [comment]
Yes - the user's context on the document definitely matters. Accessor permission level should drive what message is displayed.
